torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1827,13 +1887,26 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                         device_type='cpu',
                         dtypes=[torch.float16, torch.bfloat16]),
            )),
+    OpInfo('linalg.qr',
+           aten_name='linalg_qr',
+           op=torch.linalg.qr,
+           dtypes=floating_and_complex_types(),
+           test_inplace_grad=False,
+           # TODO: TypeError: empty_like(): argument 'input' (position 1) must be Tensor, not torch.return_types.linalg_qr
+           supports_tensor_out=False,
+           sample_inputs_func=sample_inputs_linalg_qr,
+           decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack],
+           skips=(
+               # cuda gradchecks are slow
+               # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
+               SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),)),
     OpInfo('linalg.slogdet',
            aten_name='linalg_slogdet',
            op=torch.linalg.slogdet,
            dtypes=floating_and_complex_types(),
            test_inplace_grad=False,
            supports_tensor_out=False,
-           sample_inputs_func=sample_inputs_slogdet,
+           sample_inputs_func=sample_inputs_linalg_inv,
```

Comment:
Is there a better name for sample_inputs_linalg_inv if it's being used for slogdet and pinv, too?

Reply:
I changed it to sample_inputs_linalg_invertible.